naomi/diagnose.py

```diff
@@ -102,3 +103,40 @@ def get_git_revision():
         logger.warning("Couldn't detect git revision (not a git repository?)")
         return None
     return output
+
+
+# The following were added to be able to use the user's pocketsphinx
+# settings for testing instead of requiring that a model be included
+# with the Naomi source just for testing
+def check_option_in_profile(path):
```

Comment:
While this is minor, these two methods are very similar in function. I believe in DRY code, I wonder if there is a better way of combing the two? E.G. Pass in an arg to select what you want for the key error part. A boolean would do. 

```
get_profile_val_option(path, True) ...
....
if True then its an option, else its profile option check.
```

Reply:
Sounds good. Thank you for the explanation.